### ISSUE TITLE
anaconda broke python

### ISSUE THREAD

**Issue by yggdrasil75** [rbbrdckybk/dream-factory#26]
automatic1111 requires python 3.10.9 this project uses anaconda, which is python 3.9. attempting to run automatic1111 manually (ie: to train a model) after installing this fails because it now complains that "xformers is incompatible with 3.9" please switch the project to use pip instead.

**Comment by rbbrdckybk** [comment]
Are you trying to start Auto1111 from within Dream Factory's Anaconda environment? Don't do that if so - Auto1111 sets up its own virtual environment; you should run Auto1111 from outside Anaconda (and it'll use its own separate Python instance).

I often train my own DB models using Auto1111 with the --xformers option on a machine that also has Dream Factory installed and don't have any issues with the 2 environments existing separately.

**Comment by yggdrasil75** [comment]
I had a primary automatic1111 instance which I used with this during setup. it ran, but I realized that I dont need some of the slower extensions such as dreambooth when I am just using this to generate images so I went to add a second instance. when I added the second instance, it pulled anacondas install of python (which it ships with 3.9) and refused to load xformers. I had to delete the venv and manually set it to the python 3.10 install I already had with everything working for it to load the second instance. I have since completely removed anaconda, and dreamfactory still works perfectly fine running locally on python 3.10 without an anaconda virtual environment. anaconda overwrote all of my existing python configs with its own, and I am had to pull up filehistory to get them all back even after I ran the anaconda uninstaller.
